gprotorch/dataloader/mol_prop.py

```diff
@@ -81,94 +84,51 @@ def featurize(
         :type nBits: int
         """
 
-        # auxiliary function to calculate the fingerprint representation of a molecule
```

Comment:
Looks like a sensible refactor of the DataLoader, I just thought I'd highlight @leojklarner here since he wrote the original class in case he can see potential side effects that I can't!

Reply:
Yes, thanks @bojana-rankovic for cleaning things up. I don't see any issues with this.